### PR TITLE
Update html template

### PR DESCRIPTION
**Pull request by talentwebdev**

### CODE REVIEW

File: templates/jrnl-template.html [hunk 1]
```diff
@@ -224,36 +252,35 @@ <h3 class="subtitle right-align-text">{{ item.name }}</h3>
...
-      {% for journal in item.data %}
-        <div class="two-margin journal-title-text">{{ journal.title }}</div>
-        <div class="two-margin journal-text">
-          {{ journal.content|safe }}
-        </div>
-        {% for entry in journal.entries %}
-          <h3 class="subtitle right-align-text">{{ entry.entry_date }}</h3>
-          <div class="two-margin journal-title-text">{{ entry.title }}</div>
-          <div class="two-margin journal-text">
-            {% if json_obj.photos is true %}
```
Comment (knyazz) [inline templates/jrnl-template.html:247]: turn this one back

File: templates/jrnl-template.html [hunk 2]
```diff
@@ -192,30 +219,31 @@ <h3 class="subtitle right-align-text">{{ item.name }}</h3>
...
     {% for entry in item.data.aam %}
       {% if loop.first or loop.previtem.group != entry.group %}
-        <div class="two-margin journal-title-text">{{ entry.group }}</div>
+        <span class="two-margin aam-category-text">{{ entry.group }}</span>
       {% endif %}
-      <div class="two-margin journal-text">{{ entry.title }}</div>
+      <div class="two-margin aam-question-text">{{ entry.title }}</div>
       <div class="two-margin journal-text">{{ entry.entry_date }}</div>
       <div class="two-margin journal-text" style="padding-bottom: 20mm;">
-        {% if json_obj.photos is true %}
```
Comment (knyazz) [inline templates/jrnl-template.html:214]: turn this one back

File: templates/jrnl-template.html [hunk 3]
```diff
@@ -139,51 +150,67 @@ <h5 class="document-title">
...
     <div class="table-of-contents">
       <h3>Table of Contents</h3>
         {% if json_obj.table_of_contents.intro %}
           {% set item = json_obj.table_of_contents.intro %}
-          <div class="table-of-contents-body">{{ item.name }}</div>
-          <div class="table-of-contents-body">1</div>
+          <div class="table-of-contents-body">
+            {{ item.name }}
+            </br>
+            1
```
Comment (knyazz) [inline templates/jrnl-template.html:169]: this value should be calculated

File: templates/jrnl-template.html [hunk 4]
```diff
@@ -139,51 +150,67 @@ <h5 class="document-title">
...
+        </div>
         {% endif %}
         {% if json_obj.table_of_contents.aam %}
           {% set item = json_obj.table_of_contents.aam %}
-          <div class="table-of-contents-body">{{ item.name }}</div>
-          <div class="table-of-contents-body">2</div>
+          <div class="table-of-contents-body">
+            {{ item.name }}
+            </br>
+            2
```
Comment (knyazz) [inline templates/jrnl-template.html:177]: this value should be calculated

File: templates/jrnl-template.html [hunk 5]
```diff
@@ -139,51 +150,67 @@ <h5 class="document-title">
...
+          </div>
         {% endif %}
         {% if json_obj.table_of_contents.journals %}
           {% set item = json_obj.table_of_contents.journals %}
-          <div class="table-of-contents-body">{{ item.name }}</div>
-          <div class="table-of-contents-body">3</div>
+          <div class="table-of-contents-body">
+            {{ item.name }}
+            </br>
+            3
```
Comment (knyazz) [inline templates/jrnl-template.html:185]: this value should be calculated

File: static/demo/test.html [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+    <div class='content' style="position: relative; top: 50%; transform: translateY(-50%);">
+      <h5 class="document-title">
+        My JRNL cover
+      </h5>
+      <p class="document-date">
+      
+        custom subtitle
+      
+      </p>
+      <img src="/Users/phoenix/Documents/Projects/jrnl/PDF-builder/static/images/timeline-images/logo.png" class="center logo-img" alt="logo"/>
```
Comment (knyazz) [inline static/demo/test.html:152]: change to related path, please

File: templates/jrnl-template.html (hunk 1 above)
Comment (talentwebdev) [reply templates/jrnl-template.html:247]: That code is below, Evgenii. Can you please check it once more?

**Pull request merged**